common/flatpak-run-wayland.c

```diff
@@ -246,21 +254,20 @@ flatpak_run_add_wayland_args (FlatpakBwrap *bwrap,
   struct stat statbuf;
   int fd;
 
+  wayland_display = get_wayland_display_name ();
+
 #ifdef ENABLE_WAYLAND_SECURITY_CONTEXT
   gboolean security_context_available = FALSE;
```

Comment:
This now matches code and declarations.

Reply:
Ah, but I see what you mean: there was actually no code between the first and second sets of declarations before.